### PR TITLE
fix: bump go version

### PR DESCRIPTION
**Pull request by Scartiloffista**
## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### Checklist

- [ ] Linter passes correctly
- [ ] Add tests which fail without the change (if possible)
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by mga-chka** [comment]
@Scartiloffista when do you plan to continue the PR?

**Comment by Scartiloffista** [comment]
@mga-chka, sorry, i forgot about this one. honestly idk, i thought it would have been just a version bump but there are quite some failing parts. i'll try to make space in my workload

File: .github/workflows/go-test.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.19
+        go-version: 1.22
```
Comment (vfoucault) [inline .github/workflows/go-test.yaml:19]:
```suggestion
        go-version: 1.22.2
```

**Review by vfoucault — APPROVED** [review]
LGTM

**Pull request merged**